.gitignore

```diff
@@ -45,7 +45,7 @@ src/ansi-c/gcc_builtin_headers_tm.inc
 src/ansi-c/gcc_builtin_headers_mips.inc
 src/ansi-c/gcc_builtin_headers_power.inc
 src/ansi-c/gcc_builtin_headers_ubsan.inc
-src/java_bytecode/java_core_models.inc
+src/java_bytecode/library/java_core_models.inc
```

Comment:
For future reference: please put those changes in a separate PR as it is trivial to merge (and would avoid others accidentally coming up with the same).

Reply:
Acknowledged, will do this next time!